### DISCUSSION TITLE
Welcome to sso-wall-of-shame Discussions!

### DISCUSSION THREAD

**Discussion by daallgeier** [stopthessotax/sso-wall-of-shame#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mbainter** [comment]
One idea that I saw multiple times was that of adding in companies who are doing the right thing.  I think this could be a good motivator for people fixing this problem -- especially if we were able to group those together somehow.  This might be difficult to keep up with, but we could theoretically group vendors in the same space together, plainly marking out the ones that do and do not charge the tax.

**Comment by daallgeier** [comment]
Definitely something I'd like to explore too.  Up until their acquisition Intello had its Friends of SSO list but that does not exist anymore.  

Would love to get one started.

**Comment by glodeneye** [comment]
How/where can we contribute to the list?
Here's one to add

https://featureupvote.com/pricing/
